pallets/registrar/src/tests.rs

```diff
@@ -599,7 +717,6 @@ fn weights_assigned_to_extrinsics_are_correct() {
                 .get_dispatch_info()
                 .weight,
             <() as crate::weights::WeightInfo>::mark_valid_for_collating(
-                <Test as crate::Config>::MaxGenesisDataSize::get(),
                 <Test as crate::Config>::MaxLengthParaIds::get()
             )
         );
```

Comment:
You can also add the case of `crate::Call::<Test>::pause_container_chain` to this test. Its purpose is to detect copy-paste errors, because when you add a new extrinsic you will probably copy the weight annotation from another extrinsic, so this is to ensure that you don't forget to change it.